ansible/roles/invoker/tasks/deploy.yml

```diff
@@ -31,6 +31,24 @@
     linuxOptions: "-v /usr/lib/x86_64-linux-gnu/libapparmor.so.1:/usr/lib/x86_64-linux-gnu/libapparmor.so.1"
   when: whisk_version_name != "mac"
 
+- name: get running invoker information
+  shell: >
+        curl -Gs http://{{ inventory_hostname }}:{{ docker.port }}/containers/json --data-urlencode 'filters={"name":[ "invoker" ]}'
+  register: results
```

Comment:
I have tried it and it worked.
But we need ugly url encoded due to filters.

```
- name: determine if any invokers are running
  uri: url="http://{{ inventory_hostname }}:{{ docker.port }}/containers/json?filters=%7B%22name%22:%5B%22invoker%22%5D%7D" return_content=yes
  register: invokerInfo
``` 
I couldn`t find any option to encode curly braces and brackets.

If possible, we can get rid of annoying ansible warning.